position_joint_trajectory_controller:
  type: "position_controllers/JointTrajectoryController"
  joints:
    - franka_emika_joint1
    - franka_emika_joint2
    - franka_emika_joint3
    - franka_emika_joint4
    - franka_emika_joint5
    - franka_emika_joint6
    - franka_emika_joint7
  constraints:
    goal_time: 0.01

effort_joint_trajectory_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - franka_emika_joint1
    - franka_emika_joint2
    - franka_emika_joint3
    - franka_emika_joint4
    - franka_emika_joint5
    - franka_emika_joint6
    - franka_emika_joint7
  constraints:
    goal_time: 0.01
  gains:
    franka_emika_joint1: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint2: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint3: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint4: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint5: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint6: {p: 100, d: 10, i: 0, i_clamp: 1}
    franka_emika_joint7: {p: 100, d: 10, i: 0, i_clamp: 1}

franka_state_controller:
  type: franka_hw/FrankaStateController
  publish_rate: 30  # [Hz]

